studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnType.tsx

```diff
@@ -25,17 +25,23 @@ const ColumnType: FC<Props> = ({
   showLabel = true,
   onOptionSelect = () => {},
 }) => {
-  const isNativeDataType = POSTGRES_DATA_TYPES.includes(value)
+  // @ts-ignore
+  const availableTypes = POSTGRES_DATA_TYPES.concat(enumTypes.map((type) => type.name))
+  const isAvailableType = value ? availableTypes.includes(value) : true
```

Comment:
We could consider using a `Set` here for better perf, but I doubt many users would have enough types/enums to make it make any noticeable difference.